CHANGELOG.md

```diff
@@ -8,6 +8,35 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.4.0] - 2023-03-15
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.7...v0.4.0)
+
+### Removed
+
+* [BREAKING] feat!: disable schema 1.0 support, except if appropriate flags are set (openfga/openfga#613)
+  * As of this release, OpenFGA no longer allows writing or evaluating schema `v1.0` models by default, if you need support for it for now, you can use the:
+    * `OPENFGA_ALLOW_WRITING_1_0_MODELS`: set to `true` to allow `WriteAuthorizationModel` to accept schema `v1.0` models.
+    * `OPENFGA_ALLOW_EVALUATING_1_0_MODELS`: set to `true` to allow `Check`, `Expand`, `ListObjects`, `Write` and `WriteAssertions` that target schema `v1.0` models.
+    * `ReadAuthorizationModel`, `ReadAuthorizationModels` and `ReadAssertions` are unaffected and will continue to work regardless of the target model schema version.
+  * Note that these flags will be removed and support fully dropped in a future release. Read the [Schema v1.0 Deprecation Timeline](https://openfga.dev/docs/modeling/migrating/migrating-schema-1-1#deprecation-timeline) for more details.
+
+### Added
+* feat: add OpenFGA version command to the CLI (openfga/openfga#625)
+
+### Fixed
+
+* feat: improve the speed of Check for 1.1 models by using type restrictions (openfga/openfga#545)
```

Comment:
this is not a fix, i would put it in section "added"

Reply:
Generally, I would consider pref optimizations to be fixes. Additions would be new features that are available to use. From that angle I would consider this to be a fix (even though it's an internal enhancement)